### DISCUSSION TITLE
Request for Improved Transparency Regarding Package Compatibility with Symfony 5.4, 6.4, 7.2 and Symfony UX turbo, Live

### DISCUSSION THREAD

**Discussion by hechmi2chelbi** [symfony/ux#2468] [Questions & Answers]
Hello Symfony Community,

First, I would like to express my gratitude for the incredible work you do to improve the framework and its related libraries.

I have a suggestion regarding providing clearer information about package compatibility with Symfony versions and Symfony UX, especially for packages related to Turbo, Mercure, and other associated tools. At times, it can be challenging to determine compatible versions without manual trial or extensive research in package repositories.

My proposal is to create a centralized method or an official tool that allows us to:

Easily access compatibility information for various packages with Symfony and Symfony UX versions.
Obtain a list of all Symfony UX-related packages and their compatibility with new features (e.g., Turbo and Mercure).
Include a section in the official Symfony documentation for each version, detailing compatibility (e.g., a compatibility table).
This feature would significantly help developers focus on building applications rather than spending time searching for compatibility details.

Thank you for your continuous efforts, and I hope this suggestion will be considered. If there are ongoing discussions about this topic, I kindly ask you to share them with me.

Best regards,

**Comment by smnandre** [comment]
Hi @hechmi2chelbi and thank you for your message, 

Some clarification first:

* With Webpack: the dependency requirements are specified in the various packages.json
* With AssetMapper, you do not have to bother with it, it works out the box
* Each Symfony UX package has a CHANGELOG that lists new features.
* The features available are the one on the documentation

> At times, it can be challenging to determine compatible versions without manual trial or extensive research in package repositories.

AFAIK, there’s no version of Turbo or Stimulus that is incompatible with StimulusBundle or TurboBundle. 

> Symfony versions and Symfony UX, especially for packages related to Turbo, Mercure, and other associated tools

StimulusBundle and TurboBundle follow semantic versioning and adhere to Symfony’s BC promise. Since they are stable, no compatibility or features have been removed. 

>  This feature would significantly help developers focus on building applications rather than spending time searching for compatibility details.

I haven’t heard of any compatibility errors so far.

Could you maybe provide more details about the issue you’ve encountered, to understand what happened and how we can prevent it for other users?
